src/prefect/cli/worker.py

```diff
@@ -101,6 +101,20 @@ async def start(
         case_sensitive=False,
     ),
 ):
+    """ "
+    Checks if a work-pool is paused or not
+    """
```

Comment:
This doc string looks like it belongs on the `_check_work_pool_paused` function.